File: V6.6.py
# @cikey fc86fd6b8ea7c6291f0829f791a2dfa4
# @sid 20251174010007
# @aid V6.6
import random

num = random.randint(1, 100)
tent_total = 3
tent = 0

print("Tente adivinhar o numero entre 1 e 100. Voce tem 3 tentativas.")

while tent < tent_total:
    palp = int(input(f"Tentativa {tent + 1}: "))

    tent += 1

    if palp == num:
        print(f"Parabens! Voce acertou, o numero era: {num}")
        break
    elif palp < num:
        print("O numero e maior.")
    else:
        print("O numero e menor.")

    if tent == tent_total:
        print(f"Suas tentativas acabaram. O numero era: {num}")

File: V66.java
import java.util.Scanner;
import java.util.Random;

public class V66 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        Random rand = new Random();

        int num = rand.nextInt(100) + 1;
        int tent_total = 3;
        int tent = 0;

        System.out.println("Tente adivinhar o numero entre 1 e 100. Voce tem 3 tentativas.");

        while (tent < tent_total) {
            System.out.println("Tentativa: " + (tent + 1));
            int palp = entrada.nextInt();

            tent++;

            if (palp == num) {
                System.out.println("Parabens! Voce acertou, o numero era: ");
            }
        }
        entrada.close();
    }
}   
